### DISCUSSION TITLE
[BUG] Add-PnPNavigationNode -OpenInNewTab parameter not working

### DISCUSSION THREAD

**Discussion by MelvisBass** [pnp/powershell#3208] [General]
Using the 2.1.77-nightly build, when adding -OpenInNewTab parameter to Add-PnPNavigationNode no errors received, and the node is added to menu but it does not have Open in new tab set. 

code example:
Add-PnPNavigationNode -Title $mmTitle -Location "TopNavigationBar" -Url $mmUrl -Parent $mmParentNode -PreviousNode $mmPrevNode -AudienceIds $mmAudID -OpenInNewTab

**Comment by JuergenRB** [comment]
I have created a PullRequest #3969 that solves this problem. I hope it will be merged soon
